lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -3327,6 +3315,126 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
+                            ]
+                        },
+                    "operation":
+                        { "burn" :
+                            { "quantity": 50000
+                            , "unit": "assets"
+                            }
+                        }
+                }]
+            }|]
+
+        burnAssetsCheck ctx wa tokenName' payloadBurn scriptUsed
+
+    it "TRANS_NEW_CREATE_10h - Minting assets without timelock to foreign address" $
+        \ctx -> runResourceT $ do
+        wa <- fixtureWallet ctx
+        wForeign <- emptyWallet ctx
+        addrs <- listAddresses @n ctx wForeign
+        let destination = (addrs !! 1) ^. #id
+
+        let (Right tokenName') = mkTokenName "ab12"
+
+        let payload = Json [json|{
+                "mint_burn": [{
+                    "policy_script_template":
+                        { "all":
+                           [ "cosigner#0"
+                           ]
+                        },
+                    "asset_name": #{toText tokenName'},
+                    "operation":
+                        { "mint" :
+                              { "receiving_address": #{destination},
+                                 "amount": {
+                                     "quantity": 50000,
+                                     "unit": "assets"
+                                  }
+                              }
+                        }
+                }]
+            }|]
+
+        let scriptUsed policyKeyHash = RequireAllOf
+                [ RequireSignatureOf policyKeyHash
+                ]
+
+        (_initialBalance, _expectedFee, tokens') <-
+            mintAssetsCheckWithoutBalanceCheck ctx wa tokenName' payload scriptUsed
+
+        let _minutxo = (minUTxOValue (_mainEra ctx) :: Natural)
+
+        eventually
+            "Wallet balance is decreased by fee and does not hold minted assets" $ do
+            rWa <- request @ApiWallet ctx
+                (Link.getWallet @'Shelley wa) Default Empty
+            verify rWa
+                [ expectSuccess
+                --, expectField
+                --        (#balance . #available . #getQuantity)
+                --        (`shouldBe` initialBalance - fromIntegral expectedFee - minutxo)
+                , expectField (#assets . #available . #getApiT)
+                        (`shouldBe` TokenMap.empty)
+                , expectField (#assets . #total . #getApiT)
+                        (`shouldBe` TokenMap.empty)
+                ]
+
+        eventually
+            "Foreign Wallet is initial and holds minted assets" $ do
+            rForeign <- request @ApiWallet ctx
+                (Link.getWallet @'Shelley wForeign) Default Empty
+            verify rForeign
+                [ expectSuccess
+                --, expectField
+                --        (#balance . #available . #getQuantity)
+                --        (`shouldBe` minutxo)
+                -- expected: 999978 (minutxo)
```

Comment:
this one I don't understand at this moment @jonathanknowles

Reply:
It's probably related to the minUTxO value increasing when the output contains assets. It's 344820 lovelace greater, which divided by 34482 `lovelacePerUTxOWord` is 10 words or 80 bytes, which sounds like it could be the size of the assets in the output.

Reply:
thanks Johannes! @Anviking  indeed, minimum utxo needs to be adjusted here. Now, everything is good!